os-bpmsuite-executionserver/added/launch/bpmsuite-executionserver.sh

```diff
@@ -66,6 +78,29 @@ function configure_controller_access {
     fi
 }
 
+function configure_router_access {
+    local routerService="${KIE_SERVER_ROUTER_SERVICE}"
+    routerService=${routerService^^}
+    routerService=${routerService//-/_}
+    # host
+    local kieServerRouterHost="${KIE_SERVER_ROUTER_HOST}"
+    if [ "${kieServerRouterHost}" = "" ]; then
```

Comment:
Ah, sorry, you are right. This would not work `[${kieServerRouterHost} = ""]`, but the current one will (the difference is the quotes).

Yes, I guess unifying this would not be a bad idea.